src/js/actions/transform.js

```diff
@@ -533,6 +538,9 @@ define(function (require, exports) {
             })
             .then(function () {
                 return this.transfer(layerActions.resetIndex, document);
+            })
+            .then(function () {
```

Comment:
Good catch, it might not matter but it's probably safer to end it in the correct place for future proofing.